pallets/crowdloans/src/lib.rs

```diff
@@ -643,28 +692,31 @@ pub mod pallet {
             let contributions =
                 Self::contribution_iterator(vault.trie_index, ChildStorageKind::Pending);
             let mut migrated_count = 0u32;
+            let mut non_migrated_count = 0u32;
             let mut all_migrated = true;
 
+            // single migration has a processing limit
             for (who, (amount, _)) in contributions {
                 if migrated_count >= T::MigrateKeysLimit::get() {
                     all_migrated = false;
-                    break;
+                    non_migrated_count += 1;
```

Comment:
non_migrated_count = contributions.len() - migrated_count

Reply:
`contributions` is not an array, it also doesn't have a method like `.len()`

Reply:
we have `.count` I think